feeds/backend.php

```diff
@@ -97,7 +124,7 @@
 
     if ($content == "news") {
         $data = '';
-        $result = mysqli_query(DPDatabase::get_connection(), "SELECT * FROM news_items ORDER BY date_posted DESC LIMIT 10");
```

Comment:
Where is the button for this feed? I forgot to add a `link` tag for it in my previous PR.

Reply:
Yeah, it's for all of the round pages. That query was never updated when the news stuff got updated over the years so it's showing things that are hidden/disabled and for all languages. The code that is commented out in `site_news.inc` isn't exactly useful because the feed (as coded) is showing news for the entire site, whereas the per-round page should show just those for the round (well, `news_page_id` specifically).

I'm partial to removing the news feeds entirely since getting them working correctly with the user's language, etc is probably not worth the effort for a feature that has been disabled for probably a decade or more.

Reply:
Should I remove the news stuff in `backend.php` or should we discuss it further and leave it to a future PR?

Reply:
The referencing code in site_news.inc has been commented out since before 2005 which makes it 13-year-old dead code and we should just remove it.